test/e2e/runlabel_test.go

```diff
@@ -105,7 +105,8 @@ var _ = Describe("podman container runlabel", func() {
 	})
 
 	It("podman container runlabel global options", func() {
-		Skip("Test nonfunctional for podman-in-podman testing")
+		fmt.Printf("FIXME: for lint. Remove when you fix this test: %s", GlobalDockerfile)
+		Skip("FIXME FIXME FIXME: $GLOBAL_OPTS does not work at all")
```

Comment:
Another big FIXME that needs attention. @vrothberg is this your bailiwick? FWIW I tried for a long time on the command line to make `$GLOBAL_OPTS` work, including instrumenting the podman code, and I'm pretty sure it has _never_ worked.

Reply:
Thanks, I will do so. I used your name (in vain, perhaps?) because of the git history but I didn't consider file moves.